docs/DEVELOPMENT.md

```diff
@@ -35,6 +34,30 @@ npx tauri dev
 
 Automatic recompiling on save is enabled for both the Rust and the Typescript/Vue code.
 
+If you want to build FlightCore from source, run
+
+```sh
+npx tauri build
+```
+
+This will build the executable and bundles, such as `AppImage`, `.deb` or `.msi`.
+
+To build just the executable, edit [tauri.conf.json](https://github.com/R2NorthstarTools/FlightCore/blob/main/src-tauri/tauri.conf.json) in the same folder:
+
+```json
+    "bundle": {
+      "active": true,
+```
+
+Change active from `true` to `false`, and bundles won't be included afterward.
```

Comment:
```suggestion
Change `active` from `true` to `false`, and bundles won't be included afterwards.
```

Reply:
Afterward/afterwards are both correct, just more used depending on the country, USA/UK etc.